### ISSUE TITLE
maxwidth wraps arguments inside string interpolation

### ISSUE THREAD

**Issue by ChristopherRogers** [nicklockwood/SwiftFormat#504]
Example: 

```swift
"2345678901\(String(describing: "Hello"))"
```

becomes

```swift
"2345678901\(String(
    describing: "Hello"
))"
```

with `--maxlength 20 --wraparguments beforefirst`. This doesn't compile.

**Comment by nicklockwood** [comment]
Yep, just spotted this myself. I'll push a fix asap.

**Comment by nicklockwood** [comment]
@ChristopherRogers fixed in 0.41.1
